ci/matrix.yaml

```
@@ -1,5 +1,6 @@
 
 cuda_oldest: &cuda_oldest '11.1'
+cuda_pre12:  &cuda_pre12  '11.8'
 cuda_newest: &cuda_newest '12.3'
```

I like the naming Paul uses in the devcontainer matrix. 
```suggestion
cuda_prev_min: &cuda_prev_min '11.1'
cuda_prev_max:  &cuda_prev_max  '11.8'
cuda_curr: &cuda_curr '12.3'
```